Lib/test/test_exceptions.py

```diff
@@ -292,7 +292,8 @@ def test_WindowsError(self):
             w = OSError(9, 'foo', 'bar')
             self.assertEqual(w.errno, 9)
             self.assertEqual(w.winerror, None)
-            self.assertEqual(str(w), "[Errno 9] foo: 'bar'")
+            self.assertEqual(str(w),
+                f"[Errno {errno.errorcode[errno.EBADF]}] foo: 'bar'")
```

Comment:
Why didn't you just use `"[Errno EBADF] foo: 'bar'"` instead of getting "EBADF" from the mapping?

Reply:
Hmm...